ONSdigital/babbage
# Adds pdf-merger alternative to ghostscript

### What

Replaces `ghostcript` with `PDFbox` when merging PDFs, and removes references to `ghostcript`

We need to remove it in order to bump the Java version

### How to review

- Check that the code makes sense
- Ensure all references to `ghostcript` have been removed

#### Try it locally:
You'll need to run Florence, Zebedee, dp-api-router and dp-frontend-router. As well as Babbage (make sure you checkout this branch first)

1. Login to Florence locally and create a collection
2. Start creating new page of type Bulletin
3. Edit any content you wish (it's not necessary). But do scroll down to `PDF table` and add a pdf file
4. Go through the submit, review and approve process
5. Preview the page. You should be able to download a PDF from it. The PDF should contain the contents of the page AND the contents of the file you uploaded.

### Who can review

Anyone
